VARIABLE.md

```diff
@@ -0,0 +1,128 @@
+# VARIABLES
+
+Variable is used to label and store data in computer memory.
+Variable can update information stored in variable,
+reference or get information stored in a variable.
+
+Its important to distinguish that variable are not value, they contain value and represent them.
+
+`Let` and `const` are new keyword to declare a variable, introduced in `ES6`. Prior to that programmers can declare variable with only the `var` keyword. example:
+
+```js
+var food = 'pizza';
+console.log(food); // output: pizza
+```
+
+`let` keyword signal that values can be reassigned a different value e.g
```

Comment:
`var` also can be reassigned.

Reply:
`var` can also be assigned, I forgot to include that.